Gemfile

```diff
@@ -36,6 +36,10 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'devise'
+
```

Comment:
there's really no reason to have each gem separated by the newline :)

Reply:
Yes - sometimes I still go overboard with the _neat_ and _tidy_ look. Thanks, removed the newlines.